src/explore-education-statistics-common/src/modules/table-tool/components/DataTableCaption.tsx

```diff
@@ -1,27 +1,26 @@
-import commaList from '@common/lib/utils/string/commaList';
-import {
-  LocationFilter,
-  TimePeriodFilter,
-} from '@common/modules/full-table/types/filters';
-import React from 'react';
-import { FullTableMeta } from '@common/modules/full-table/types/fullTable';
 import ButtonText from '@common/components/ButtonText';
-import classNames from 'classnames';
 import useToggle from '@common/hooks/useToggle';
+import commaList from '@common/lib/utils/string/commaList';
+import { FullTableMeta } from '@common/modules/table-tool/types/fullTable';
+import classNames from 'classnames';
+import React from 'react';
 
 export function generateTableTitle({
+  indicators,
   timePeriodRange,
   locations,
   subjectName,
   publicationName,
   expanded,
-}: {
-  timePeriodRange: FullTableMeta['timePeriodRange'];
-  locations: FullTableMeta['locations'];
-  subjectName: FullTableMeta['subjectName'];
-  publicationName: FullTableMeta['publicationName'];
+}: FullTableMeta & {
   expanded?: boolean;
 }) {
+  let indicatorString = '';
```

Comment:
`const indicatorString = indicators.length === 1 ? `${indicators[0].label} for ` : ''`?